### PR TITLE
feat(*): Add functionality to enumerate references list

### PR DESCRIPTION
**Pull request by GeekyShacklebolt**
This PR does as follows:

- Add python script to modify `main.bbl` to enumerate references list

- Add makefile to run shell commands to generate the pdf.

- Add instructions in readme for using makefile.

- Add hanging reference indentation in `main-style.tex` to give ` 0.5-inch` indentation to the second line (wrapped line) onwards of references.

- Update sample.pdf and put it in a separate directory.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,23 +29,43 @@ Install [LaTex](https://www.latex-project.org/get/). If you are on Linux, then y
...
+    pdflatex main
+    pdflatex main
 ```
 
 __NOTE__:
-- Every time you add a new reference, you need to run these 3 commands.
-- *.aux* file is generated automatically after using `pdflatex` for the initial build.
+- Every time you add/remove a reference you need to re-generate the PDF. For which either run command `make` as described [above](#method-1-use-the-makefile), OR do as follows:
+```shell
+    rm main.bbl
```
Comment (CuriousLearner) [inline README.md]: You can just mention this one command.

Also, I don't think it is necessary to remove these files if I want to regenerate PDF according to the new changes, isn't it?

File: {{cookiecutter.project_name}}/makefile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+jdf:
```
Comment (CuriousLearner) [inline {{cookiecutter.project_name}}/makefile]: Also, define the `PHONY_LIST` in the `Makefile`

File: README.md (hunk 1 above)
Comment (GeekyShacklebolt) [reply README.md]: Yes, you are right, we don't need them. I'll change.

File: {{cookiecutter.project_name}}/makefile [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.PHONY: all
+all:
+	$(info Generating the JDF PDF.)
+	pdflatex main
```
Comment (CuriousLearner) [inline {{cookiecutter.project_name}}/makefile]: Shouldn't this be `main.tex`?

Likewise in all other commands?

Comment (GeekyShacklebolt) [reply {{cookiecutter.project_name}}/makefile]: It auto detects that.

Comment (CuriousLearner) [reply {{cookiecutter.project_name}}/makefile]: Sounds good. Thanks!

**Comment by CuriousLearner** [comment]
I've tested this on local as well. Looks good. But I really think we should change the color of the link to keep it consistent.

I really like the `#2EA3F2` for links. It looks pleasant to the eyes. If you've time, feel free to do another PR for this reference link.

I'm merging this one for now :)

Thanks for all your work!

**Pull request merged**